src/runtime/mini_webgpu.h

```diff
@@ -31,21 +31,21 @@
 #define WEBGPU_H_
 
 #if defined(WGPU_SHARED_LIBRARY)
-#    if defined(_WIN32)
```

Comment:
This is unrelated formatting change (ok to keep here, just wanted to make sure)

Reply:
No, I'll revert, for some reason my local clang-format isn't reading the .clang-format file correctly